iks_manual_steps.md

```diff
@@ -0,0 +1,100 @@
+# IKS Manual Steps
+This document outlines the manual steps that are needed to connect IBM Cloud Internet Services (CIS) and IBM Kubernetes Service
+
+## 1. Adding Domain Name to CIS Instance
+Within the CIS console in the **Overview** tab add your domain name and then configure DNS records with the ingress subdomain present on your IKS Cluster. DNS records are optional during this step and can be done later in the **Reliability** tab in CIS. In our situation two CNAME DNS records were added to CIS named `@` and `www` with each of their values being the ingress subdomain from the IKS Cluster wanting to be connected.
```

Comment:
I don't think we should put here that the DNS records are optional this time, since we definitely need them for IKS

Reply:
Maybe I need to reword this then. I mainly meant that the DNS records are optional when adding your domain to CIS but will need to be done at some point.

Reply:
maybe just in the sentence that starts with "DNS records are optional..." at the end we add "...but must be completed to connect the IKS cluster to CIS"